apps/website/screens/theme-generator/components/previews/DataGrid.tsx

```diff
@@ -60,6 +80,7 @@ const expandableRows = [
     priority: "High",
     issueType: "Epic",
     expandedContent: <DxcContainer>Custom content 2</DxcContainer>,
+    actions: <DxcDataGrid.ActionsCell actions={actions as any} />,
```

Comment:
What do you think about exporting the type `Action` in `packages/lib/src/table/types.ts` 

```
export type ActionCellsPropsType = {
  actions: Actions;
};

export type Actions = Array<
  | {
      icon: string | SVG;
      title: string;
      onClick: () => void;
      disabled?: boolean;
      tabIndex?: number;
      options?: never;
    }
  | {
      icon?: never;
      title: string;
      onClick: (value?: string) => void;
      disabled?: boolean;
      tabIndex?: number;
      options: Option[];
    }
>;
```

This way, we could prevent using `actions as any` everywhere, including `Table.stories` and `ResultsetTable.stories`

Reply:
I added the export, but I don't think there will be any change in this file.